api/HTMLElement.json

```diff
@@ -935,6 +935,172 @@
           }
         }
       },
+      "drag_event": {
+        "__compat": {
+          "description": "<code>drag</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLElement/drag_event",
+          "spec_url": "https://html.spec.whatwg.org/multipage/indices.html#ix-handler-ondrag",
```

Comment:
Can you add https://html.spec.whatwg.org/multipage/dnd.html#event-dnd-drag and similar to spec URLs?